### PR TITLE
Tokenisers for Japanese/Chinese

### PR DESCRIPTION
**Pull request by donomii**
### What's being changed:

Add tokenisers for japanese and chinese

Also adds a "DEFAULT_TOKENIZER" environment variable, that will be used to set the tokeniser for a property when it is automatically created by loading an object

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: adapters/repos/db/inverted/bm25_searcher.go [hunk 1]
```diff
@@ -119,6 +119,8 @@ func (b *BM25Searcher) wand(
 		models.PropertyTokenizationLowercase,
 		models.PropertyTokenizationWhitespace,
 		models.PropertyTokenizationField,
+		models.PropertyTokenizationTrigram,
+		models.PropertyTokenizationGse,
 	}
```
Comment (dirkkul) [inline adapters/repos/db/inverted/bm25_searcher.go:124]: can we have a list of all tokenizers in the Tokenizer package and use that everywhere? Then it is easier to to not forget adding a new one

**Review by dirkkul — DISMISSED** [review]
Can't say much about the new tokenizers function, but code looks good :)

File: adapters/repos/db/inverted/bm25_searcher.go (hunk 1 above)
Comment (donomii) [reply adapters/repos/db/inverted/bm25_searcher.go:124]: Sounds like a great idea, where should we put the list?

Comment (dirkkul) [reply adapters/repos/db/inverted/bm25_searcher.go:124]: I think the adapters/repos/db/helpers/tokenizer.go file is a natural place for this and you already defining the list there anyways

File: adapters/repos/db/helpers/tokenizer.go [hunk 2]
```diff
@@ -91,6 +155,17 @@ func tokenizeWordWithWildcards(in string) []string {
 	return lowercase(terms)
 }
 
+// tokenizetrigram splits on any non-alphanumerical and lowercases the words, joins them together, then groups them into trigrams
+func tokenizetrigramWithWildcards(in string) []string {
```
Comment (parkerduckworth) [inline adapters/repos/db/helpers/tokenizer.go:159]: Was this comment accidentally lifted from the `tokenizetrigram` function header? Looks the same, and this comment doesn't explain the wildcard part

Comment (donomii) [reply adapters/repos/db/helpers/tokenizer.go:159]: yes, thanks for spotting that

**Review by dirkkul — APPROVED** [review]

**Pull request merged**